src/status_im/ui/screens/desktop/views.cljs

```diff
@@ -10,12 +10,26 @@
             [status-im.ui.screens.accounts.create.views :as create.views]
             [status-im.ui.screens.accounts.login.views :as login.views]
             [status-im.ui.screens.accounts.recover.views :as recover.views]
-            [status-im.ui.screens.accounts.views :as accounts.views]))
+            [status-im.ui.screens.accounts.views :as accounts.views]
+            [status-im.utils.platform :as platform]
+            [status-im.i18n :as i18n]
+            [status-im.react-native.js-dependencies :as rn-dependencies]
+            [taoensso.timbre :as log]
+            [status-im.utils.utils :as utils]))
 
 (enable-console-print!)
 
 (views/defview main []
-  (views/letsubs [view-id [:get :view-id]]
+  (views/letsubs [view-id [:get :view-id]
+                  version [:get-app-version]]
+    {:component-did-mount
+     (fn []
+       (when platform/desktop?
```

Comment:
this view only used in desktop so you don't need this check

Reply:
good point, thanks!